### ISSUE TITLE
Feature Request: taskbar agent application

### ISSUE THREAD

**Issue by technoicon** [wh1te909/tacticalrmm#253]
I know we spoke about it but thought I would make a feature request.

Would be nice to have a tray icon/taskbar icon that runs when the agent installs.

Some of the main features I would require are:
-Custom Logo.
-When hovering Icon, show computer name.
-Be able to set custom menu items, some examples:
      A clickable URL - say to launch website or ticketing system.
      A way to load a page (so you could say, load a chat window or embed a iframe in a window).
      Display some text.
      Download a file.
      Run a script.

Thanks.

**Comment by dinger1986** [comment]
Yes I agree this would be brilliant. I mentioned it before as well and the issue is everything is running as a system service so getting comms to the user can be hard with that (from my understanding).

Would be cool to have it though!

**Comment by bradhawkins85** [comment]
RunAsUser powershell will help with getting things from system to user level

https://github.com/KelvinTegelaar/RunAsUser

Add BurntToast to RunAsUser for notification to the user

https://www.cyberdrain.com/monitoring-with-powershell-notifying-users-of-windows-updates/

**Comment by dinger1986** [comment]
I thought I had seen something like that at some point. Wouldn't be hard with that I guess.

**Comment by technoicon** [comment]
Currently I found this:
 https://code.google.com/archive/p/traylauncher/

and I was able to add a link to chrome with a website launch and also add just text, so phone number etc.

There is a go app on git that can be made to sit in the system tray, not sure if that would help?

**Comment by Shootify** [comment]
yes, please add this whenever possible.

**Comment by georgebarnick** [comment]
Another nice menu item to include would be a ticket submission form button. Previous RMMs I've worked with had this functionality which made it easy to direct end users to creating tickets with any PSA or helpdesk software. Basically just clicking the menu option would open up a form that had fields for Name, Email, Phone, and Description, as well as the ability to attach or take a screenshot, and then when they submit that it emails to the configured help desk email address.

**Comment by silversword411** [comment]
I think this is fundamentally requesting a systray icon for TRMM that would be user configurable. This looks like a good starting point: https://github.com/getlantern/systray

**Comment by JSuenram** [comment]
+1

**Comment by plao** [comment]
+1 for me also!

**Comment by framatt** [comment]
+1

**Comment by stavros-k** [comment]
Please don't +1... you can use reactions to show that you like the said feature.

+1 is just spamming all the people following this feature request and adds zero value or ideas to that.

Thanks

**Comment by screwlooseit** [comment]
I'd love a feature like a trmm tray icon to be implemented into Tactical where a user can right-click on the tray Icon and be met with the following options

Identify My PC - This gives them an output of their hostname
Submit a support request - This gives them a text box where they can type in an issue they are experiencing and it then either sends an email to the helpdesk or generates an alert in the dashboard.
Show Support contact - This is a field that can be edited showcasing an email and phone number or whatever is deemed necessary
Go to the Support website - a button that opens a web browser to a user-defined website

I know there is an alternative below. But it would be really nice if this was built into the agent or a feature you could enable for agents.
https://github.com/conlan0/Trayicon 

+1 would love to have this feature
